index.json

```diff
@@ -0,0 +1,112 @@
+[
```

Comment:
Given on the image building stage we use the index generator to generate the index.json on the fly, I am wondering do we still need the index.json file in this repo?

Reply:
Oops, that definitely was not supposed to get committed. Will remove 😅

Reply:
👍